vehicle/mkvi/software/brakes/bspd.h

```diff
@@ -0,0 +1,38 @@
+#include "libs/adc/api.h"
+#include "libs/gpio/api.h"
+#include "libs/gpio/pin_defs.h"
+#include "libs/timer/api.h"
+
+//////////////////////// GPIO - DIGITAL ////////////////////////
+// These are digital outputs for avr-controlled LED outputs
+gpio_t MOTOR_5KW_LED = PB0;
+gpio_t BRAKE_LL_LED = PB1;
+gpio_t BSPD_TRIP_LED = PB2;
+
+// Monitor Pins connected to the logic-level (LL) side of the LSDs
+gpio_t BSPD_LL = PB3;
+gpio_t BRAKELIGHT_LL = PB4;
+
+// Digital sense pin for 5kW motor "on" state
+gpio_t MOTOR_CURRENT_SENSE = PB5;
+
+// Input for shutdown sense line
+// gpio_t BSPD_SHUTDOWN_SENSE = PC0;
+
+//////////////////////// GPIO - ANALOG /////////////////////////
```

Comment:
nit: the GPIO usually refers exclusively to digital pins, so analog inputs aren't part of that